pypeit/pypeit.py

```diff
@@ -656,7 +657,17 @@ def reduce_one(self, frames, det, bg_frames, std_outfile=None):
                             slits=True, clear=True)
 
         # Prep for manual extraction (if requested)
-        manual_extract_dict = self.fitstbl.get_manual_extract(frames, det)
+        #manual_extract_dict = self.fitstbl.get_manual_extract(frames, det)
+        # TODO:
+        #  Object finding, this appears inevitable for the moment, since we need to be able to call find_objects
+        #  outside of reduce. I think the solution here is to create a method in reduce for that performs the modified
+        #  2d coadd reduce
+        if self.par['reduce']['extraction']['manual']['spat_spec'] is not None:
+            spats, specs, dets, fwhms = extract.parse_manual(self.par['reduce']['extraction']['manual'])
+            manual_extract_dict = dict(hand_extract_spec=specs, hand_extract_spat=spats,
+                                     hand_extract_det=dets, hand_extract_fwhm=fwhms)
```

Comment:
You're taking what is effectively a dictionary (a `ParSet`), parsing it, and then creating a separate dictionary with an undefined data model.  Please don't do this.

Reply:
cleaned up (a bit)